cjwkernel/types.py

```diff
@@ -792,6 +806,16 @@ def to_dict(self) -> Dict[str, Any]:
             "quickFixes": [qf.to_dict() for qf in self.quick_fixes],
         }
 
+    def to_pandas_error(self):
```

Comment:
Nix this function and implement something in `cjwkernel.pandas.types` instead. `ProcessError.from_arrow()` and `ProcessError.to_arrow()` would make sense.

The `cjwkernel.types` module is (perhaps ill-advisedly) aware of Thrift; but it is not aware of Pandas.